profiles/api.py

```diff
@@ -38,3 +45,20 @@ def get_channels(user):
             )
         )
     )
+
+
+def get_site_type_from_url(url):
+    """
+    Gets a site type (as defined in profiles.models) from the given URL
+
+    Args:
+        url (str): A URL
+
+    Returns:
+        str: A string indicating the site type
+    """
+    no_fetch_extract = tldextract.TLDExtract(suffix_list_urls=False)
```

Comment:
[This uses the TLD snapshot](https://github.com/john-kurkowski/tldextract#note-about-caching), right?  Seems like the safest option since it shouldn't change much.

Reply:
I'm passing in `suffix_list_urls=False` here because (1) I got permission errors in my docker containers when the library was trying to write to that cache file, and (2) my tests were failing because the library was attempting to make a live request for some information about a URL (https://github.com/john-kurkowski/tldextract/blob/master/tldextract/remote.py#L27). (2) could be solved by monkeypatching, but in general it seemed easiest just to forego the fetch and I didn't see any risks. Accurately matching a domain only matters for our social sites (facebook, twitter, linkedin), and I think we can be confident that there won't be any funky new TLDs for those sites. Could be wrong, though...